docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,512 @@
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1176f370",
+   "metadata": {},
+   "source": [
+    "### `log_state`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3547b04",
+   "metadata": {},
+   "source": [
+    "The `log_state` argument can be passed in `run_tardis()` to set the logging level for the simulation. The input for this argument **must be** one of the following: **Notset, Debug, Info, Warning, Error** or **Critical**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc2dcd71",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "sim = run_tardis(\"tardis_config_logger.yml\", log_state=\"Info\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52e778bc",
+   "metadata": {},
+   "source": [
+    "By setting up the `log_state` parameter to **\"Info\"** in the above example, we can check that the logs are at the **\"Info\"** or higher logging level. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25277fe8",
+   "metadata": {},
+   "source": [
+    "### `specific`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56a5ef36",
+   "metadata": {},
+   "source": [
+    "The `specific` argument tells the logger to capture log messages set by the `log_state` parameter. It can only take *Boolean* values for input i.e, `True` or `False`. Take for example the following :"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a0df43d1",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "sim = run_tardis(\"tardis_config_logger.yml\", log_state=\"Debug\", specific=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c6ff57c",
+   "metadata": {},
+   "source": [
+    "It can be examined that, when we set `specific` to `True`, the log messages captured were only at the `DEBUG` log level.<br> This allows for logging only specified logging messages for inspection."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5083321b",
+   "metadata": {},
+   "source": [
+    "The changes in the captured log messages can be seen when we set `specific` to `False`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "566919b3",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "sim = run_tardis(\"tardis_config_logger.yml\", log_state=\"Debug\", specific=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42ddc98b",
+   "metadata": {},
+   "source": [
+    "It can be examined in this example that when we kept `specific` to `False` then the captured log output includes all log messages from `DEBUG` and higher logging level, which is the default behavior of the logger."
```

Comment:
Slight reword. "It can be examined in this example that when we kept `specific` to `False`, the captured log output includes all log messages from `DEBUG` and higher logging levels, which is the default behavior of the logger."

Changes that I made: remove then and add a comma after `False`, and makes levels plural as there are more than one level.